docs/controls/access_control/ac-06.md

```diff
@@ -0,0 +1,16 @@
+# Access Control
+## AC-06 - Least Privilege
+
+The organization employs the principle of least privilege, allowing only authorized accesses for users (or processes acting on behalf of users) which are necessary to accomplish assigned tasks in accordance with organizational missions and business functions.
+
+AC-6 (2) Additional FedRAMP Requirements and Guidance: Examples of security functions include but are not limited to: establishing system accounts, configuring access authorizations (i.e., permissions, privileges), setting events to be audited, and setting intrusion detection parameters, system programming, system and security administration, other privileged functions.
+
+### TDP Implementation
+
+In order to access the TDP application, users must login through Login.gov, using password and MFA.  Only authorized users can access TDP.
```

Comment:
Please change "MFA" to "Multifactor Authentication (MFA)"

Reply:
Updated to Multi-factor Authentication (MFA)